i18n/en.json

```diff
@@ -3,26 +3,29 @@
   "info": "Help translate text labels on SVG files. Pick a file on commons to get started.",
   "beta": "This feature is in early testing.",
   "learn-more": "Learn more & leave feedback!",
-  "powered-by-toolforge": "Powered by Toolforge",
-  "report-a-bug": "Report a bug",
-  "privacy-policy": "Privacy policy",
-  "view-on-github": "View on Github",
-  "licenced-under": "Licenced under $1",
-  "gpl3": "GPL 3.0 or later",
-  "made-by": "Made by the Wikimedia Foundation's $1",
-  "commtech": "Community Tech team",
+
   "help": "Help",
-  "pick-an-image": "Pick an image",
-  "pick-an-image-more": "from Commons.",
-  "translate-labels": "Translate",
-  "translate-labels-more": "the labels in the image.",
-  "update-commons": "Update Commons",
-  "update-commons-more": "or download directly to your computer.",
   "login": "Log in",
   "logout": "Log out",
   "language-settings": "Language settings",
+
+  "search-help": "Type a file name from Commons",
   "search-label": "File name (without 'File:' prefix) or URL:",
   "search-placeholder": "100 Years War France 1435.svg",
   "translate-button": "Translate",
-  "search-help": "Type a file name from Commons"
+
+  "pick-an-image": "<strong>Pick an image</strong> <span>from Commons.</span>",
+  "translate-labels": "<strong>Translate</strong> <span>the labels in the image.</span>",
+  "update-commons": "<strong>Update Commons</strong> <span>or download directly to your computer.</span>",
+  "commons-logo-alt": "The Wikimedia Commons logo",
+  "translate-icon-alt": "An icon indicating multiple languages",
+  "download-icon-alt": "An icon indicating file-download",
+
+  "licenced-under": "Licenced under <a href='https://www.gnu.org/licenses/gpl-3.0.html'>GPL 3.0 or later</a>",
```

Comment:
If we word it like "Licensed under GPL 3.0+" (without _or later_) then I think we can safely inject the link as a param, "Licensed under $1", and avoid the HTML